app/views/wpcc/your_results/index.html.erb

```diff
@@ -48,12 +48,10 @@
       <% end %>
     </div>
 
-    <% if earning_below_tax_relief_threshold?(session[:salary], session[:salary_frequency]) %>
+    <% if @message_presenter.tax_relief_warning? %>
       <div class="contributions__row">
         <div class="callout">
-          <p>
-            <%= t('wpcc.results.tax_relief_warning_html') %>
-          </p>
+          <p><%= t('wpcc.results.tax_relief_warning_html') %></p>
```

Comment:
Since you're map the message in a presenter, I would move this translation to the presenter object too, something like message_presenter.tax_relief_warning_message.